master/pkg/schemas/expconf/log_pattern_config.go

```diff
@@ -40,7 +75,8 @@ func (b LogPoliciesConfigV0) Merge(
 type LogPolicyV0 struct {
 	RawPattern string `json:"pattern"`
 
-	RawAction LogActionV0 `json:"action"`
+	RawActions []LogActionV0 `json:"actions,omitempty"`
```

Comment:
did anyone explicitly ask for multiple actions? today the available actions seem mutually exclusive in terms of when you would want them to me, and this is a breaking config change for anyone using this featuer.

Reply:
> did anyone explicitly ask for multiple actions

The "signal" action shows up in the UI, but you might also want an action with a side-effect.

> this is a breaking config change

We should be implementing shims to keep from breaking old configs.

@jgongd Do you have the examples of the desired config examples that we worked out together?  I feel like @stoksc would benefit from seeing the overall goal here.

Reply:
I was concerned that it might be a breaking change as well!

Shim is implemented [here](https://github.com/determined-ai/determined/pull/9947/files#diff-b131920afae395773bce3cf4120f8186ca08c96d028a915ccfc70d380ad59bf5R163-R179).

Both legacy log policy and modern log policy are accepted now: https://github.com/determined-ai/determined/pull/9947/files#diff-ced209455ef7f0c29127fe9cc7a734f53cf0012d1268916b89afe0ed4f98ad17

Tested [here](https://github.com/determined-ai/determined/pull/9947/files#diff-e1c26ed8b923497a36fcb692a5cea7b555b2a34159efc0e49bbf9dae4cb0b11aR123).